jetstream_pt/layers.py

```diff
@@ -127,13 +134,285 @@ def repeat_kv(x: torch.Tensor, n_rep: int) -> torch.Tensor:
       .reshape(bs, n_kv_heads * n_rep, slen, head_dim)
   )
 
+DEFAULT_MASK_VALUE = -0.7 * float(np.finfo(np.dtype("float32")).max)
+
+def ragged_flash_attention_kernel(
```

Comment:
Do you consider to create a ragged_attention util class and move ragged code into that class?

Reply:
Good point! Moves the related logic to attention_kernel.py, all the dense kernel, quantized kernel, ragged attention kernel impls are there.